contracts/Zauction.sol

```diff
@@ -225,6 +392,23 @@ contract ZAuction is Initializable, OwnableUpgradeable {
     topLevelDomainFee[id] = amount;
   }
 
+  /// Get the defined ERC20 token used for payment of a domain in either bidding
+  /// or immediate buying.
+  /// @param domainId The id of the domain to get an ERC20 token for
+  function getTokenForDomain(uint256 domainId) public view returns (IERC20) {
```

Comment:
Nit: Could we rename this to `getPaymentTokenForDomain`

We are dealing with a lot of "tokens" in zAuction so it might be better to be verbose.